### DISCUSSION TITLE
Change to /home

### DISCUSSION THREAD

**Discussion by orcachillin** [EssentialsX/Essentials#4571] [Ideas and Feature Suggestions]
When using /home with no arguments, teleport the player to /home home instead of showing the list of homes.  To see the list of homes, use /homes

**Comment by seema84** [comment]
A default home is probably what most people expect when they type /home.

**Comment by pop4959** [comment]
It already does what you're saying unless you have more than one home, otherwise it shows you a list. There isn't any other way to display your home list right now IIRC.

**Comment by seema84** [comment]
I know, homes should perhaps not be an alias of home but an extra command, so that home always points to the default home (even if there is more than one).
